ColumnSet.js

```diff
@@ -59,9 +76,12 @@ define([
 		if (node.nodeType !== 1) {
 			node = node.parentNode;
 		}
-		while (node && !query.matches(node, '.dgrid-column-set[' + colsetidAttr + ']', root)) {
+
+		while (node && !isRootNode(node, root) && !query.matches(node, '.dgrid-column-set[' + colsetidAttr + ']', root))
```

Comment:
Changing this condition like this will cause this loop to bail out with a truthy `node` value when `isRootNode` returns `true`, which I would expect to cause problems.  Oddly, I only actually observe it causing errors in the touch code path, not the mousewheel code path.  I am testing with moving the `isRootNode` check inside the loop.